packages/cli/src/commands/alias/ls.ts

```diff
@@ -46,10 +45,11 @@ export default async function ls(
   const { aliases, pagination } = await getAliases(
     client,
     undefined,
-    nextTimestamp
+    validated.nextTimestamp,
+    validated.limit
```

Comment:
Using the spread operator would make it easier to add/remove future pagination related options.
```suggestion
    ...paginationOptions
```

Reply:
It does, but it returning an iterator leaves us vulnerable to someone changing the order of the iterator and breaking things, but that's probably ok.